requirements/generated/requirements-linting.txt

```diff
@@ -13,6 +13,8 @@ anyio==4.4.0
     #   httpx
     #   starlette
     #   watchfiles
+bcrypt==4.2.0
```

Comment:
Do we need these for linting?

Reply:
Yep, linting extends the base library which requires bcrypt